config/clusters/2i2c/support.values.yaml

```diff
@@ -3,6 +3,7 @@ prometheusIngressAuthSecret:
 
 prometheus:
   server:
+    retention: 180d
```

Comment:
I'm looking at this and thinking `180d` means "180 days", but the title of this PR indicates we want to save data for 1 year. So either I've misunderstood, or this is a typo?

Reply:
@sgibson91 yep, missed a push! fixed